### PR TITLE
compile at android-studio and catkin_make

### PR DESCRIPTION
**Pull request by talregev**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,26 +15,47 @@
...
+    apply from: "https://github.com/talregev/rosjava_mvn_repo/raw/master/buildscript.gradle"
+    dependencies {
+        classpath "com.android.tools.build:gradle:1.0.1"
+    }
 }
 
 apply plugin: 'catkin'
 
 allprojects {
+    apply from: "https://github.com/talregev/rosjava_mvn_repo/raw/master/maven.gradle"
```
Comment (stonier) [inline build.gradle:31]: I didn't know you could do this....thats a rather useful and visible way of outlining where things are getting sourced from.

Comment (talregev) [inline build.gradle:31]: that cool thing i learn on the way.
I added this line to all ros-java project. 
i think it the best way to do it.

re-arrange the maven_repo by the way. you can look at this.

File: build.gradle [hunk 2]
```diff
@@ -15,26 +15,47 @@
...
 task wrapper(type: Wrapper) {
-  gradleVersion = '1.11'
+    gradleVersion = '2.2.1'
 }
 
 buildscript {
-  apply from: "https://github.com/rosjava/rosjava_bootstrap/raw/indigo/buildscript.gradle"
+    apply from: "https://github.com/talregev/rosjava_mvn_repo/raw/master/buildscript.gradle"
+    dependencies {
+        classpath "com.android.tools.build:gradle:1.0.1"
```
Comment (stonier) [inline build.gradle:24]: @talregev Why adding the android dependency here? Usually this is just built as a jar, packaged into a maven artifact, and then used by android projects.

Comment (talregev) [inline build.gradle:24]: you don't need it. i just added by default. but it compile.

File: build.gradle (hunk 1 above)
Comment (stonier) [inline build.gradle:31]: Might do this for the non-api parts of the ros gradle plugin. It's an extra line, but that visibility will help people contribute when changes happen.

**Comment by stonier** [comment]
Ok, closing this. Wrapper update went in 39333110. The rest seem to not have any important effect.

The apply trick I'll try across the board in a bit.

Thanks!

**Pull request closed without merging**